Application/Frontend/pages/api/add-member.ts

```diff
@@ -0,0 +1,75 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { MongoClient } from 'mongodb';
+import { getMongoDbUri } from '@/lib/dbConfig';
+
+/**
+ * Handles the addition of a friend request between two users in the Accord application. It accepts a POST
```

Comment:
Again, these docstrings are for the friend request endpoint, not for the add-member endpoint. We are not sending friend requests here but rather adding someone else to an existing group chat.